### PR TITLE
Adding Tajpuriya data CSV file

### PR DESCRIPTION
**Pull request by Thepuja**

### CODE REVIEW

File: Tajpuriya grammer - Sheet1.csv [hunk 1]
```diff
@@ -0,0 +1,41 @@
+English,Tajpuriya/Rajbanshi,English,Tajpuriya/Rajbanshi
```
Comment (christikaes) [inline Tajpuriya grammer - Sheet1.csv:1]: Can you please remove the duplicate header - there should only be one for English and one for Tajpuriya

Comment (christikaes) [reply Tajpuriya grammer - Sheet1.csv:1]: When you do this make sure that the extra commas on lines 2-end are removed

**Review by christikaes — CHANGES_REQUESTED** [review]

File: Tajpuriya grammer - Sheet1.csv [hunk 2]
```diff
@@ -0,0 +1,41 @@
+English,Tajpuriya
+Go,जा ,
```
Comment (christikaes) [inline Tajpuriya grammer - Sheet1.csv:2]: I think there is still an extra space and comma here, it should be: `Go,जा` not `Go,जा ,` the newline is parsed as a new entry so we don't need the comma

File: Tajpuriya grammer - Sheet1.csv
Comment (christikaes) [inline Tajpuriya grammer - Sheet1.csv:1]: Can we also rename this file to `TajpuriyaGrammer.csv`?

**Review by christikaes — CHANGES_REQUESTED** [review]

File: Tajpuriya grammer - Sheet1.csv
Comment (christikaes) [reply Tajpuriya grammer - Sheet1.csv:1]: We can do this in a followup PR if you want to merge now

**Review by christikaes — APPROVED** [review]

**Comment by binamkayastha** [comment]
Merging even if the ci/cd is failing, because we know this is safe to merge in (it won't effect the ci/cd)

**Pull request merged**